CONTRIBUTING.adoc

```diff
@@ -0,0 +1,55 @@
+= Keep Contribution Guide
+
+🎉 Thanks for taking the time to contribute! 🎉 Contributions are welcome from
+anyone, and even the smallest of fixes is appreciated!
+
+The following is a set of guidelines for contributing to Keep and its packages.
+These are mostly guidelines, not rules. Use your best judgment, and feel free to
+propose changes to this document in a pull request.
+
+== Getting started
+
+1. Fork
+   https://github.com/keep-network/sortition-pools[`keep-network/sortition-pools`]
+2. Clone your fork
+3. Follow the
+   link:README.md#Setup[installation & build steps] in the README.
+4. Set up the <<Development Tooling>>.
+5. Open a PR against the `main` branch and describe the change you are intending
+   to undertake in the PR description.
+
+Before marking the PR as ready for review, make sure:
+
+* It passes the linter checks (`npm run lint` and `npm run lint:fix`).
+* It passes the https://github.com/keep-network/sortition-pools/actions[continuous
+  integration tests].
+* Your changes have sufficient test coverage (e.g regression tests have
+  been added for bug fixes, unit tests for new features).
+
+== Development Tooling
+
+Commits
+https://help.github.com/en/articles/about-commit-signature-verification[must
+be signed].
+
+=== Continuous Integration
+
+Keep uses https://github.com/keep-network/sortition-pools/actions[Github
+Actions] for continuous integration. All jobs must be green to merge a PR.
+
+=== Linting
+
+Linters and formatters for Solidity, JavaScript, and Go code are set up and run
+automatically as part of pre-commit hooks. These are checked again in CI builds
```

Comment:
It might be useful adding a paragraph on pre-commit hooks just like you did in other repos:

```
=== Pre-commit

Pre-commit is a tool to install hooks that check code before commits are made.
It can be helpful to install this, to automatically run linter checks and avoid
pushing code that will not be accepted. Follow the
https://pre-commit.com/[installation instructions here], and then run
`pre-commit install` to install the hooks.
```

Reply:
[6d724d](https://github.com/keep-network/sortition-pools/pull/202/commits/6d724d884ef87d6fb5dd96993b87b028deee8836)